src/components/Events/EventForm.vue

```diff
@@ -257,6 +277,7 @@ export default {
       if (this.validateInput()) {
         this.event.details.start = this.dateTimeToUnix(this.eventDate, this.startTime);
         this.event.details.end = this.dateTimeToUnix(this.eventDate, this.endTime);
+        this.event.price = Math.round(this.event.price * 100); // convert dollars to cents
```

Comment:
Similar conversion logic happens in multiple places in the code. Try abstracting conversion logic into a single helper method in utils that we can use in these places.